### ISSUE TITLE
Great job of collecting PWA resources!!! KUDOS!!!

### ISSUE THREAD

**Issue by inglesuniversal** [hemanth/awesome-pwa#114]
My main question for the time being would be... Out of your experience... Which tool can you recommend to build PWA from web URL's online that can work on both iOS and Android devices out of the box?

Thanks!

**Comment by maxence-charriere** [comment]
Hello there, was just passing by to ask to add my app and Saw this question.

I created a framework to build PWA with the Go programming language (Golang) name [go-app](https://github.com/hemanth/awesome-pwa).

It is pretty uncommon to code frontend with Go but this one make it possible.
I just released a PWA at the start of the week: [Lofimusic.app](https://lofimusic.app), an app that lists and plays Lo-Fi radio. Handling a music player with web tech on mobile is a little bit challenging but this one does a good enough job.
